android/fastlane/Fastfile

```diff
@@ -0,0 +1,27 @@
+fastlane_require 'pry'
+fastlane_version '2.137.0'
+import 'Fastfile.private'
+
+platform :android do |options| 
+
+  desc "Submit a new QA Release Build to Firebase App Distribution."
+  lane :distribute_qa do |options|
```

Comment:
why is distribute instead of release ? Its for follow the firebase name?

Reply:
yep, we only have one name with `release`, because it sends the apk to the playstore instead of firebase... 🤔 

maybe we can invert those names